Lesson25/Kadir/script.js

```diff
@@ -6,11 +6,37 @@ p (.post-body)
 */
 
 const URL = 'https://jsonplaceholder.typicode.com/posts';
+const postsContainer = document.getElementById('post-list');
+
 function fetchPosts() {
-}
+	fetch(URL)
+		.then((response) => response.json())
+		.then((posts) =>
+			posts.forEach((post) => {
+				const listItem = document.createElement('li');
+				listItem.classList.add('post');
+
+				const heading = document.createElement('h2');
+				heading.classList.add('post-title');
+				heading.innerText = post.title;
+				listItem.appendChild(heading);
+
+				const paragraph = document.createElement('p');
+				paragraph.classList.add('post-body');
+				paragraph.innerText = post.body;
+				listItem.appendChild(paragraph);
+
+				postsContainer.appendChild(listItem);
+			})
+		)
+    .catch((error) => {
+			console.error('An error occured: ', error);
+		});
+};
 
 function clearPosts() {
+  postsContainer.innerHTML = '';
```

Comment:
Use innerText instead of innerHTML. See this article for details: https://medium.com/@verity.carlos/why-you-shouldnt-use-innerhtml-and-what-to-use-instead-ed99d064a416

Reply:
Done